ai21/clients/studio/resources/chat/chat_completions.py

```diff
@@ -6,6 +6,9 @@
 from ai21.clients.studio.resources.chat.base_chat_completions import BaseChatCompletions
 from ai21.models import ChatMessage as J2ChatMessage
 from ai21.models.chat import ChatMessage, ChatCompletionResponse, ChatCompletionChunk
+from ai21.models.chat.document_schema import DocumentSchema
+from ai21.models.chat.response_format import ResponseFormat
```

Comment:
Please add these import to `ai21/models/chat/__init__.py` so their imports would be `from ai21.models.chat ...`

Reply:
Added